src/services/backingImage.js

```diff
@@ -28,6 +28,12 @@ export async function deleteBackingImage(params) {
   }
 }
 
+export async function download(params) {
+  if (params && params.name) {
+    window.location.href = `${ window.__pathname_prefix__ }${ window.__pathname_prefix__.endsWith('/') ? '' : '/'}v1/backingimages/${params.name}/download` // eslint-disable-line
```

Comment:
Will this open another target to download or replace the current window/tab?

Reply:
If this is a download link, it will not open a new page, the browser will download the file directly. @innobead